src/services/user.js

```diff
@@ -142,6 +199,14 @@ const userService = {
 
   deleteUser: async (id) => {
     await User.findByIdAndRemove(id).exec()
+  },
+
+  _calculateDeletionMainSubject: async (userId, categoryId) => {
+    const aggregateOptions = [{ $match: { category: categoryId, author: userId, status: OFFER_STATUS_ENUM[0] } }]
+    const userOffers = await offerService.getOffers(aggregateOptions)
+    const userCooperations = await cooperationService.getCooperations(aggregateOptions)
+
+    return !!userOffers || !!userCooperations
```

Comment:
Better use `Boolean(userOffers) || Boolean(userCooperations)` or try `Boolean(userOffers || userCooperations)`

Reply:
Thank you! fixed